### DISCUSSION TITLE
Sound Notification

### DISCUSSION THREAD

**Discussion by TheOnlyMissouri** [dailydotdev/daily#1468] [Feature Requests]
can we have sound in notification with the possibility to enable or disable sounds and adjust their volume within the app's settings and reserve sound notifications for high-priority messages or events that require immediate attention

**Comment by idoshamun** [comment]
Hi @TheOnlyMissouri,

Thanks for the suggestion. If you're interested in sound then I suggest you enable push notifications as they make a sound by default
